### DISCUSSION TITLE
Name alias for modules

### DISCUSSION THREAD

**Discussion by Eccenux** [npm/feedback#1020] [Registry]
Seems like there is no easy way to add an alias to a module, right? I think it would be good to add an alias for names that are not that obvious in spelling. I mean adding names that someone might easily mixup (e.g. dash / no-dash name).

It might look something like this in `package.json`:
```js
{
  name:"wiki-ploy",
  alias:"wikiploy",
}
```
```js
{
  name:"socket.io",
  alias:["socket-io","socketio"],
}
```

**Comment by ljharb** [comment]
No, there’s no way to do that - a package only has one name.

npm already disallows publishing when the name is too similar to an existing package.
